### PR TITLE
added note and distance operator to tips and tricks

### PR DESCRIPTION
**Pull request by csobier**
@iriberri , for issue #133 .  Thanks for clarifying the syntax!  Let me know if you sign-off and I'll merge.

### CODE REVIEW

File: cartodb-editor.md [hunk 1]
```diff
@@ -1141,7 +1141,7 @@ You can add and style a map legend for a published map. Map legends can be symbo
 
 	<p class="wrap-border"><img src="{{ '/img/layout/cartodb-editor/legend_output.png' | prepend: site.baseurl }}" alt="Example Legend on a published map" /></p>
 	
-	**Tip:** To view other beautiful maps, see the CartoDB [Gallery](https://cartodb.com/gallery/) page.
```
Comment (iriberri) [inline cartodb-editor.md]: Hahaha, ohhh, no longer beautiful :P (joking ;))

**Comment by iriberri** [comment]
Fine! Take into account using block syntax for the next time we share "long" or more complex SQL statements, but it's :+1:

File: cartodb-editor.md (hunk 1 above)
Comment (csobier) [inline cartodb-editor.md]: Still beautiful, of course, but that's a marketing buzz term, ha!

On Thu, Oct 1, 2015 at 8:22 AM, Carla <notifications@github.com> wrote:

> In cartodb-editor.md
> <https://github.com/CartoDB/docs/pull/399#discussion_r40906573>:
>
> > @@ -1141,7 +1141,7 @@ You can add and style a map legend for a published map. Map legends can be symbo
> >
> >  	<p class="wrap-border">Example Legend on a published map</p>
> >  	
> > -	**Tip:** To view other beautiful maps, see the CartoDB [Gallery](https://cartodb.com/gallery/) page.
>
> Hahaha, ohhh, no longer beautiful :P (joking ;))
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CartoDB/docs/pull/399/files#r40906573>.
>



-- 
*Christa Sobier*
Technical Writer

*CartoDB—Map Your World's Data*
143 Roebling St 2A, Brooklyn, New York, 11211, USA
T: +1.917.463.3232
W: www.cartodb.com

Live sessions to learn CartoDB <http://cartodb.com/webinars/>

**Comment by csobier** [comment]
Thanks, for the time being I kept the format the same as the other syntax.

On Thu, Oct 1, 2015 at 8:23 AM, Carla <notifications@github.com> wrote:

> Fine! Take into account using block syntax for the next time we share
> "long" or more complex SQL statements, but it's [image: :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CartoDB/docs/pull/399#issuecomment-144711153>.
>



-- 
*Christa Sobier*
Technical Writer

*CartoDB—Map Your World's Data*
143 Roebling St 2A, Brooklyn, New York, 11211, USA
T: +1.917.463.3232
W: www.cartodb.com

Live sessions to learn CartoDB <http://cartodb.com/webinars/>

**Pull request merged**